vnext/Shared/OInstance.cpp

```diff
@@ -538,35 +538,38 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
   // If this code is enabled, we will have unused module instances.
   // Also, MSRN has a different property bag mechanism incompatible with this method's transitionalProps variable.
 #if (defined(_MSC_VER) && !defined(WINRT))
-  modules.push_back(std::make_unique<CxxNativeModule>(
-      m_innerInstance,
-      Microsoft::React::GetHttpModuleName(),
-      [nativeQueue, transitionalProps]() -> std::unique_ptr<xplat::module::CxxModule> {
-        return Microsoft::React::CreateHttpModule(transitionalProps);
-      },
-      nativeQueue));
-
-  modules.push_back(std::make_unique<CxxNativeModule>(
-      m_innerInstance,
-      Microsoft::React::GetWebSocketModuleName(),
-      [nativeQueue, transitionalProps]() -> std::unique_ptr<xplat::module::CxxModule> {
-        return Microsoft::React::CreateWebSocketModule(transitionalProps);
-      },
-      nativeQueue));
-
-  // Use in case the host app provides its a non-Blob-compatilbe HTTP module.
-  if (!Microsoft::React::GetRuntimeOptionBool("Blob.DisableModule")) {
+  // Applications using the Windows ABI feature should loade the networking TurboModule variants instead.
+  if (!Microsoft::React::GetRuntimeOptionBool("UseWindowsAbi")) {
```

Comment:
UseWindowsAbi is an Office thing.  Can we name this something that specifies what it does rather than who is using it?

Reply:
Sure.